### ISSUE TITLE
Add `any()` strategy

### ISSUE THREAD

**Issue by MarcoFavorito** [HypothesisWorks/hypothesis#1690]
It would be nice to have an `any()` strategy that generates any possible object.

Probably this is already possible by using something like:
```python
from_type(type).flatmap(lambda t: from_type(t))
```

But a shortcut `any()` is far more readable.

**Comment by Zac-HD** [comment]
Well, `any` is a built-in name, so we'd have to call it something else!  

The main issue though is that functions which should accept *literally anything* are actually very rare, and it's unlikely that a very fancy implementation would be any better at finding bugs than `from_type(type).flatmap(from_type)` (once #1685 is merged, of course).  On the other hand, we would have to test, document, and maintain the code - and users would have to skip over it to find other functions that would be useful in their specific case.

I do suggest checking out the [battle_tested`](https://github.com/CodyKochmann/battle_tested) library though, as a great way to throw everything at your particular function!
